File: BeerPlease/app/src/main/java/app/dgandroid/eu/beerplease/model/Beer.java
package app.dgandroid.eu.beerplease.model;

import com.google.gson.annotations.SerializedName;
import java.io.Serializable;

/**
 * Created by Duilio on 22/05/2017.
 */

public class Beer implements Serializable {
    @SerializedName("id")
    private int id;
    @SerializedName("name")
    private String name;
    @SerializedName("tagline")
    private String tag;
    @SerializedName("description")
    private String description;
    @SerializedName("image_url")
    private String image;
    @SerializedName("contributed_by")
    private String contributorsName;
    @SerializedName("first_brewed")
    private String dateOfBrew;
    @SerializedName("ingredients")
    private Ingredients ingredients;

    public String getName() {
        return name;
    }

    public Ingredients getIngredients() {
        return ingredients;
    }

    public int getId() {
        return id;
    }

    public String getContributorsName() {
        return contributorsName;
    }

    public String getDateOfBrew() {
        return dateOfBrew;
    }

    public String getDescription() {
        return description;
    }

    public String getImage() {
        return image;
    }

    public String getTag() {
        return tag;
    }
}

File: BeerPlease/app/src/main/java/app/dgandroid/eu/beerplease/model/Beer.kt
package app.dgandroid.eu.beerplease.model

import com.google.gson.annotations.SerializedName
import java.io.Serializable

/**
 * Created by Duilio on 22/05/2017.
 */

class Beer : Serializable {
    @SerializedName("id")
    val id: Int = 0
    @SerializedName("name")
    val name: String = ""
    @SerializedName("tagline")
    val tag: String? = null
    @SerializedName("description")
    val description: String = ""
    @SerializedName("image_url")
    val image: String = ""
    @SerializedName("contributed_by")
    val contributorsName: String = ""
    @SerializedName("first_brewed")
    val dateOfBrew: String = ""
    @SerializedName("ingredients")
    val ingredients = Ingredients()
}